src/i18n/gr/index.js

```diff
@@ -2,55 +2,55 @@ export default {
         },
-        usePostingKey: '����� �������� ���������',
-        experimental: '�����������',
+        usePostingKey: 'Χρήση Κλειδιού Ανάρτησης',
+        experimental: 'πειραματικό',
         steemPostingKey: {
-          label: '������ ���������',
-          hint: '�� �������������� �� ����� ������ ���!'
+          label: 'Κλειδί Ανάρτησης',
+          hint: 'Μη χρησιμοποιείτε το κύριο κλειδί σας!'
         },
         gitRepo: {
-          label: '���������� GIT',
+          label: 'Αποθετήριο GIT',
           hint: 'https://github.com/g-u-c/guc-desktop'
         },
         workingDirectory: {
-          label: '������� ��������',
-          hint: '��� ��������� �� ���� ���� ���������� ���;'
+          label: 'Φάκελος Εργασίας',
+          hint: 'Που βρίσκεται το έργο στον υπολογιστή σας;'
         },
         commitId: {
           label: 'Commit ID',
           hint: 'C0FF33'
         }
       },
       edit: {
-        label: '�����������',
-        tags: '��������',
-        postTitle: '������ ���������'
+        label: 'Επεξεργασία',
+        tags: 'Ετικέτες',
+        postTitle: 'Τίτλος Ανάρτησης'
       },
       review: {
-        label: '����������'
+        label: 'Αναθεώρηση'
       },
       inform: {
-        label: '���������',
-        p1: '�� Git-Utopian-Contributor (G-U-C) ����� ��� �������� ��� ������������� �� ����� ��� �������� ��� �������� ������������ ��� ��������������� ��� ������ �� ������������� �� ������� ���� �� ��� ���������� ��� Steem ��� ��� Utopian.',
-        p2: '�� ���� �����, � �������� ��� ��������� �� ���������� ������ (� ��� ���� ������ ����� �� ��������� ������) ��� ��� ���������� ��� GIT. �� �������� �� ����������� �� ������� ��� ��� GIT ��� ���� �� ������������ ��� ������� ��� ��� blockchain ��� Steem. � �������� ����� �������� ��� ��������������� ������� �� ��� ��������� ��� ��������� ���������� �������� ������.',
-        p3: '������������� ��� ������� ��� Utopian Hackathon, ����������� 2018.<br><br>����� MIT.<br>�����: nothingismagick, DrSensor, decebal'
+        label: 'Ενημέρωση',
+        p1: 'Το Git-Utopian-Contributor (G-U-C) είναι ένα εργαλείο που δημιουργήθηκε με σκοπό την ενίσχυση της εργασίας δημοσιεύσεων των προγραμματιστών που θέλουν να διαμοιραστούν τη δουλειά τους με τις κοινότητες του Steem και του Utopian.',
+        p2: 'Με λίγα λόγια, η εφαρμογή σας επιτρέπει να προσθέτετε σχόλια (ή και κατα κάποιο τρόπο να αφαιρείτε σχόλια) απο τις σημειώσεις του GIT. Θα μπορείτε να καταγράφετε τη δουλειά σας στο GIT και μετά να δημοσιεύσετε την δουλειά σας στο blockchain του Steem. Η ανάρτηση είναι δομημένη και ετικετοποιημένη σύμφωνα με την κατηγορία της ανάπτυξης λογισμικού ανοιχτού κώδικα.',
+        p3: 'Δημιουργήθηκε στα πλαίσια του Utopian Hackathon, Σεπτέμβριος 2018.<br><br>’δεια MIT.<br>Ομάδα: nothingismagick, DrSensor, decebal'
       }
     },
     dashboard: {
-      title: '����������'
+      title: 'Επισκόπηση'
```

Comment:
Seems like the de-fonts written in before has different encoding with my system :sweat_smile: 
I use Ubuntu 18.04